bindings_test.go

```diff
@@ -32,3 +33,84 @@ func TestCreateDataChunk(t *testing.T) {
 	chunk := CreateDataChunk(types)
 	defer DestroyDataChunk(&chunk)
 }
+
+func TestArrow(t *testing.T) {
+	defer VerifyAllocationCounters()
+
+	var config Config
+	defer DestroyConfig(&config)
+	if CreateConfig(&config) == StateError {
+		t.Fail()
+	}
+
+	var db Database
+	defer Close(&db)
+	var errMsg string
+	if OpenExt(":memory:", &db, config, &errMsg) == StateError {
```

Comment:
We can likely simplify this to check for `StateSuccess` - and empty error is still unexpected.